contributors/devel/sig-release/release.md

```diff
@@ -1,8 +1,8 @@
-# Targeting Features, Issues and PRs to Release Milestones
+# Targeting enhancements, Issues and PRs to Release Milestones
```

Comment:
Not a blocker to getting this merged, but potentially a follow-up item. Throughout the doc there is still a lot of mixed use of feature / enhancement and the images still reference feature freeze instead of enhancement freeze. It might be worth going through and cleaning up the language a bit more.

Reply:
@mrbobbytables -- yeah, I think that doc needs a full rewrite when we do enhancements doc cleanups. The goal here was just "make it a little less inaccurate". :sweat_smile:

Reply:
![](https://i.imgur.com/LWRuEYM.gif)
/lgtm